ci/release/update-version.sh

```diff
@@ -17,11 +17,13 @@ CURRENT_MAJOR=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[1]}')
 CURRENT_MINOR=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[2]}')
 CURRENT_PATCH=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[3]}')
 CURRENT_SHORT_TAG=${CURRENT_MAJOR}.${CURRENT_MINOR}
+CURRENT_UCXPY_VERSION=$(curl -s https://version.gpuci.io/rapids/${CURRENT_SHORT_TAG}`.*)
```

Comment:
Is this needed?

Reply:
Not currently, I had it for completeness but I removed now since it's unused at this time.